actors/builtin/miner/miner_actor.go

```diff
@@ -1998,6 +2023,22 @@ func resolveOwnerAddress(rt Runtime, raw addr.Address) addr.Address {
 	return resolved
 }
 
+// Resolves a control address to an ID address and verifies that it is address of an account actor.
+func resolveControlAddress(rt Runtime, raw addr.Address) addr.Address {
```

Comment:
Rename `resolveOwnerAddress` to `resolveControlAddress` and use that. I don't see any reason to exclude multisigs here; that could even be useful.